src/dlgIRC.cpp

```diff
@@ -1,6 +1,8 @@
 /***************************************************************************
+ *   Copyright (C) 2008-2017 The Communi Project                           *
  *   Copyright (C) 2008-2013 by Heiko Koehn - KoehnHeiko@googlemail.com    *
  *   Copyright (C) 2014 by Ahmed Charles - acharles@outlook.com            *
+ *   Copyright (C) 2017 by Fae - itsthefae@gmail.com                       *
```

Comment:
Ah, I found some more information that clarifies the matter a bit in an answer on [Opensource SE](https://opensource.stackexchange.com/a/4036/3722) - basically you can (at least) in the USA use a pseudonym in your Copyright line - it just doesn't last as long...